src/main/resources/db/migration/V23__health_offer.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE health_offer (
+    id                  bigserial,
+    user_id             text,
+    title               text,
+    description         text,
+    mode                text,
+    specialization      text,
+    city                text,
+    region              text,
+    language            text,
+    modified_date       timestamp not null,
+    user_first_name     text,
+    status              text default 'ACTIVE',
+    phone_number        text,
+    phone_country_code  text,
+    CONSTRAINT PK_HCO_ID PRIMARY KEY (id)
+);
+
+CREATE TABLE health_offer_AUD (
+    id                  bigserial,
+    user_id             text,
+    title               text,
+    description         text,
+    mode                text,
+    specialization      text,
+    city                text,
+    region              text,
+    language            text,
+    modified_date       timestamp not null,
+    user_first_name     text,
+    status              text default 'ACTIVE',
+    phone_number        text,
+    phone_country_code  text,
+    REV                 integer not null,
+    REVTYPE             smallint,
+    REVEND              integer,
+    PRIMARY KEY (id, REV),
+    CONSTRAINT FK_health_offer_AUD_REV FOREIGN KEY (REV) REFERENCES revinfo (REV),
+    CONSTRAINT FK_health_offer_AUD_REVEND FOREIGN KEY (REVEND) REFERENCES revinfo (REV)
+);
+
+CREATE INDEX idx_health_offer_city ON health_offer (upper(city));
+CREATE INDEX idx_health_offer_region ON health_offer (upper(region));
+CREATE INDEX idx_health_offer_status ON health_offer USING hash(status);
+CREATE INDEX idx_health_offer_user_id ON health_offer USING hash(user_id);
+CREATE INDEX idx_health_offer_modified_date ON health_offer (modified_date);
+CREATE INDEX idx_health_offer_specialization ON health_offer (specialization);
```

Comment:
Missing indexes for `mode` and `language`?
In order for index to work for `... like '%value%'` queries, there might be a need for `text_pattern_ops` index (see `V19__change_city_index.sql` file).

But I would suggest to check manually if this index is actually applied - you can do this by [enabling logging of SQL statements](https://www.baeldung.com/sql-logging-spring-boot) and using [Postgres explain](https://www.postgresql.org/docs/current/sql-explain.html) syntax.

Reply:
After discuss we decided to merge it. Indexes will be added in future.